.github/workflows/actions-test.yml

```diff
@@ -6,14 +6,16 @@ jobs:
     strategy:
       matrix:
         node: [12.13.1]
-    container:
-      image: node:${{ matrix.node }}
     steps:
       - name: Check out repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node }}
```

Comment:
I would probably remove the matrix at this point and just put the version here. But that will change your required checks.